src/plugins/biostruct3d_view/biostruct3d_view.pri

```diff
@@ -1,20 +1,5 @@
 # include (biostruct3d_view.pri)
 
-defineTest( use_deprecated_view ) {
```

Comment:
Restore the deprecated view. It is still used for Qt versions less than 5.4. The minimum supported Qt version is 5.2.1.